projects/batfish/src/main/java/org/batfish/main/Batfish.java

```diff
@@ -4118,16 +4147,11 @@ private void serializeHostConfigs(Path testRigPath, Path outputPath,
                   hostConfig.getIptablesFile());
             String relativePathStr = _testrigSettings.getBasePath()
                   .relativize(path).toString();
-            if (!iptablesConfigurations.containsKey(relativePathStr)) {
-               for (String key : iptablesConfigurations.keySet()) {
-                  _logger.errorf("key : %s\n", key);
-               }
-               throw new BatfishException(
-                     "Key not found for iptables: " + relativePathStr);
+            if (iptablesConfigurations.containsKey(relativePathStr)) {
```

Comment:
I am not sure if the iptable file failed, should I set the parse_status of the host file to be failed? It was success.